geometry_msgs/Point[4] rect

